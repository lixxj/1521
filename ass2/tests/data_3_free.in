a = malloc 100
b = malloc 200
c = malloc 300
d = malloc 400
e = malloc 500
free b
free d
free c