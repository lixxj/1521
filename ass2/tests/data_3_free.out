+00000 (F, 5000) 
[a] +00008 
+00000 (A,  108) +00108 (F, 4892) 
[a] +00008 [b] +00116 
+00000 (A,  108) +00108 (A,  208) +00316 (F, 4684) 
[a] +00008 [b] +00116 [c] +00324 
+00000 (A,  108) +00108 (A,  208) +00316 (A,  308) +00624 (F, 4376) 
[a] +00008 [b] +00116 [c] +00324 [d] +00632 
+00000 (A,  108) +00108 (A,  208) +00316 (A,  308) +00624 (A,  408) +01032 (F, 3968)
[a] +00008 [b] +00116 [c] +00324 [d] +00632 [e] +01040 
+00000 (A,  108) +00108 (A,  208) +00316 (A,  308) +00624 (A,  408) +01032 (A,  508)
+01540 (F, 3460) 
[a] +00008 [c] +00324 [d] +00632 [e] +01040 
+00000 (A,  108) +00108 (F,  208) +00316 (A,  308) +00624 (A,  408) +01032 (A,  508)
+01540 (F, 3460) 
[a] +00008 [c] +00324 [e] +01040 
+00000 (A,  108) +00108 (F,  208) +00316 (A,  308) +00624 (F,  408) +01032 (A,  508)
+01540 (F, 3460) 
[a] +00008 [e] +01040 
+00000 (A,  108) +00108 (F,  924) +01032 (A,  508) +01540 (F, 3460) 
+00000 (A,  108) +00108 (F,  924) +01032 (A,  508) +01540 (F, 3460) 
